AutomatedTesting/Gem/PythonTests/AWS/aws_metrics/aws_metrics_utils.py

```diff
@@ -224,6 +224,22 @@ def delete_table(self, database_name: str, table_name: str) -> None:
             Name=table_name
         )
 
+    def try_delete_table(self, database_name: str, table_name: str) -> None:
```

Comment:
I think we can get rid of the delete_table function if it's not used by the test code any more:

https://github.com/o3de/o3de/blob/development/AutomatedTesting/Gem/PythonTests/AWS/aws_metrics/aws_metrics_utils.py#L214

Reply:
Hmm, good call, will see its still used and remove if its now been abandoned.